packages/amplify-cli/src/initialize-env.js

```diff
@@ -63,15 +63,16 @@ async function initializeEnv(context, currentAmplifyMeta) {
     );
     await sequential(initializationTasks);
 
+    spinner.succeed(
+      isPulling ? `Successfully pulled backend environment ${currentEnv} from the cloud.` : 'Initialized provider successfully.',
+    );
+    // if pull is successful further errors would result from initializing the env
+    isPulling = false;
```

Comment:
Since isPulling will always be false, we can remove these lines and the subsequent checks below

```js
    ...
    // Generate AWS exports/configurtion file
    await context.amplify.onCategoryOutputsChange(context, currentAmplifyMeta);
    context.print.success('Initialized your environment successfully.');
  } catch (e) {
    spinner.fail('There was an error initializing your environment.');
    throw e;
  }
```

Reply:
It can still fail while running `await sequential(initializationTasks);` hence the pull message. Though I figure since the code here does the same thing on a pull or init we can change the error message to be the same as you suggested.

Reply:
Ah that's an oversight by me. I guess it's just the context.print.success() call that can be simplified. Edited the above comment.

Reply:
Changed the fail message to be the same. We can remove the `isPulling` as well if we think the message would be more concise to just say it's initializing the env.